easybuild/easyconfigs/s/Salmon/Salmon-1.4.0-GCC-11.2.0.eb

```diff
@@ -0,0 +1,61 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au>
+# License::     GLPv2
+#
+# Notes::
+##
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+# Thanks to people like Simon Brandord, Mikael Oehmann and Kenneth Hoste
+# for their help with the ICE
+
+easyblock = 'CMakeMake'
+
+name = 'Salmon'
+version = '1.4.0'
+
+homepage = 'https://github.com/COMBINE-lab/salmon'
+description = """Salmon is a wicked-fast program to produce a highly-accurate,
+ transcript-level quantification estimates from RNA-seq data."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
```

Comment:
@wdlingit could you say why you've changed from `gompi` to `GCC`, and removed the `openmp` build option?

Reply:
@verdurin Here is one rather personal reason: the openmpi is not really integrated with my inhouse slurm cluster so that I currently got no test space to build it with openmpi. Another reason is that, I checked its install instructions and found no requirement of openmpi. This is why I changed it from gompi to gcc. Anyway, this EB was create to make #15419 runnable, at least to me. Please feel free to make any changes if needed.

Reply:
@wdlingit it's certainly fine to adjust it as a dependency of something else. Just wanted to make sure it was done intentionally. Thanks for the explanation.